lighthouse-core/gather/driver.js

```diff
@@ -918,6 +911,9 @@ class Driver {
           maxWaitMs);
     }
 
+    // Awaiting so that errors can be properly processed.
```

Comment:
nit: while this comment makes total sense in the context of this PR, I can definitely see myself coming back in a year and being like "what?" maybe a link to this PR or a tad more explanation?

Reply:
good call. Maybe even just something like `// Bring any 'Page.navigate' errors back into the promise chain.` and a link back here.

The comment above the `Page.navigate` command isn't terribly helpful either, but I don't have a great suggestion :)